tests/sse/test_sse_undefined.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+// This file uses SSE and SSE2 by calling undefined functions and prints the results.
+// Use a diff tool to compare the results between platforms.
+
+#include <stdio.h>
+#include <inttypes.h>
+#include <float.h>
+
+#ifdef __EMSCRIPTEN__
+#include <emscripten/emscripten.h>
+#endif
+
+#ifndef __cplusplus
+#include <stdalign.h>
+#endif
+
+#include <emmintrin.h>
+#define ENABLE_SSE2
+
+void test_undefined_sse()
+{
+	// SSE Undefined instructions:
+	#ifdef __EMSCRIPTEN__
+	__m128 ps = _mm_undefined();
+	#else
+	__m128 ps = _mm_undefined_ps();
+	#endif
```

Comment:
Oh, hmm, this is an interesting find. Not quite sure why this difference. https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_undefin lists only `_mm_undefined_ps` and no `_mm_undefined`. 

Feel free to also add an implementation of `_mm_undefined_ps` to xmmintrin.h (in addition to `_mm_undefined`) if you'd like to simplify this.